pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/HealthCheck.kt

```diff
@@ -85,8 +102,12 @@ class HealthQueryService: KoinComponent {
     fun getHealth(): HealthCheck {
```

Comment:
sonar cloud tags this function with `brain overload`, I agree it's too much for a single function. I will refactor with private functions for checking health.